e2e/testsuite/testsuite.go

```diff
@@ -313,6 +314,14 @@ func (s *E2ETestSuite) getRelayerExecReporter() *testreporter.RelayerExecReporte
 	return rep.RelayerExecReporter(s.T())
 }
 
+// GetTimeoutHeight returns a timeout height of 1000 blocks above the current block height.
+// This function should be used when the timeout is never expected to be reached
+func (s *E2ETestSuite) GetTimeoutHeight(ctx context.Context, chain *cosmos.CosmosChain) clienttypes.Height {
```

Comment:
I opened an issue to move this to the cosmos chain https://github.com/strangelove-ventures/ibctest/issues/240